primer/brand
# Add mona to the footer logo

## Summary

Adds Mona to the Footer Logo

## List of notable changes:

- **Updated** `MinimalFooter`

## What should reviewers focus on?

- Is the spacing between the svgs correct?
- Was the correct mark used?

## Steps to test:

1. Open up the `MinimalFooter` stories on storybook
1. Ensure that the Footer appears correctly in all modes

## Supporting resources (related issues, external links, etc):

- Fixes [github/primer #2632](https://github.com/github/primer/issues/2632)

## Contributor checklist:


## Reviewer checklist:


## Screenshots:

> Please try to provide before and after screenshots or videos

<table>
<tr>
<th> Before</th> <th> After </th>
</tr>
<tr>
<td valign="top">


![CleanShot 2023-10-25 at 09 19 24@2x](https://github.com/primer/brand/assets/12260694/2dd39eba-303a-4a10-bb0e-eb9a445028b4)

 </td>
<td valign="top">


![CleanShot 2023-10-25 at 09 20 05@2x](https://github.com/primer/brand/assets/12260694/ba29676e-0359-4197-877b-737dd770aa4c)

</td>
</tr>
</table>
